layout/column.ftd

```diff
@@ -17,6 +17,12 @@ integer border-bottom: $ds.borders.none
 integer border-left: $ds.borders.none
 integer border-right: $ds.borders.none
```

Comment:
Also these are wrong. If I want to specify border on any of the `ds.row, column` etc, I should only specify one property, `border`, not this 4. `border: $ds.borders.uniform.medum` (you want same on all sides), or `border: `ds.borders.top.medium` (you want only one border), or `border: ds.borders.vertical.medium` (which applies only left and right borders).

Reply:
```ftd
-- component foo:
border-data border:

-- record border-data:
integer left:
integer right:
integer top:
integer bottom:


-- record border-thickness:
border-data medium:
border-data large:
...


-- border-thickness uniform:

-- uniform.medium:
top: 10
bottom: 10
right: 10
left: 10

-- uniform.medium:
top: 20
bottom: 20
right: 20
left: 20

-- end: uniform




-- border-thickness vertical:

-- vertical.medium:
top: 10
bottom: 10
right: 0
left: 0

-- vertical.medium:
top: 20
bottom: 20
right: 0
left: 0

-- end: vertical



-- border-thickness top:

-- top.medium:
top: 10
bottom: 0
right: 0
left: 0

-- top.medium:
top: 20
bottom: 0
right: 0
left: 0

-- end: top
```